src/assets/README.md

```diff
@@ -1,100 +1,6 @@
-```
-
-## Eleventy Integration
-
-Laravel Mix generates an asset manifest in JSON format (this example shows the assets with a [version string for cache
-busting](https://laravel-mix.com/docs/5.0/versioning)):
-
-```json
-{
-    "/scripts/app.js": "/scripts/app.js?id=f45bfe43b5e6cb80a6da",
-    "/styles/main.css": "/styles/app.css?id=bc8ec84445a00d932c8a",
-    "/scripts/matomo.js": "/scripts/matomo.js?id=3155c9bb3c237aa64c27",
-    "/scripts/uio.js": "/scripts/uio.js?id=41d589185ccf95cc14f9"
-}
-```
-
-The [`scripts.njk`](../_includes/partials/scripts.njk) and [`stylesheets.njk`](../_includes/partials/stylesheets.njk)
-template partials use this asset manifest to load scripts and stylesheets in the site header. When running webpack and
-Eleventy in watch mode (using `npm start`), a successful webpack build will update the asset manifest, triggering an
-Eleventy build and refreshing the local development site with updated CSS and JavaScript assets.
-
-If you wish to load different scripts or stylesheets on different pages, you will need to modify the
-[`scripts.njk`](../_includes/partials/scripts.njk) and [`stylesheets.njk`](../_includes/partials/stylesheets.njk)
-template partials to accomplish this.
-
-## Advanced Methods
-
-### Automatic Processing
-
-The Mix [configuration file](../../webpack.mix.js) can be modified to automatically process all JavaScript, CSS, or
-Sass files in the root of the [`/src/assets/scripts`](scripts) or [`/src/assets/styles`](styles) directory as
-follows.
-
-1. Add the [`fast-glob`](https://www.npmjs.com/package/fast-glob) package to your devDependencies:
-
-    ```bash
-    npm i --save-dev --save-exact fast-glob
-    ```
-
-2. Require `fast-glob` in the Mix [configuration file](../../webpack.mix.js):
-
-    ```diff
-    const mix = require("laravel-mix");
-    + const fg = require("fast-glob");
-    const moveFile = require("move-file");
-    ```
-
-3. Replace the relevant section (for JavaScript, CSS and/or Sass — this example shows JavaScript):
-
-    ```diff
-    // Process JavaScript files with Babel.
-    - mix.js("./src/assets/scripts/app.js", "dist/assets/scripts");
-    - mix.js("./src/assets/scripts/matomo.js", "dist/assets/scripts");
-    - mix.js("./src/assets/scripts/uio.js", "dist/assets/scripts");
-    + const scriptPaths = fg.sync("./src/assets/scripts/*.js");
-    +
-    + scriptPaths.forEach(entryPath => {
-    +     mix.js(entryPath, "dist/assets/scripts");
-    + });
-    ```
-
-This technique [globs](https://en.wikipedia.org/wiki/Glob_(programming)) all files of the specified type and then calls
-the appropriate Mix function against each file in the resulting array.
+The ILDH uses [eleventy-plugin-fluid](https://github.com/fluid-project/eleventy-plugin-fluid) to process JavaScript and CSS or [Sass](http://sass-lang.com) stylesheets. For more information, see the [`eleventy-plugin-fluid` docs](https://github.com/fluid-project/eleventy-plugin-fluid#asset-handling).
 
 #### Caveats
```

Comment:
This whole section is mostly inaccurate now and should be removed. Same with Automatic Loading below.